dev/provider_packages/PROVIDER__INIT__PY_TEMPLATE.py.jinja2

```diff
@@ -44,9 +44,9 @@ from __future__ import annotations
 
 import packaging.version
 
-__all__ = ["version"]
+__all__ = ["__version__"]
 
-version = "{{ RELEASE }}{{ VERSION_SUFFIX }}"
+__version__ = "{{ RELEASE }}{{ VERSION_SUFFIX }}"
```

Comment:
Just so this isn't a breaking change for all providers, should we have both?

Reply:
These haven't been in any release yet I don't think

Reply:
Coreect